test/SmokeTests/OOProcSmokeTests/DotNetIsolated/Patterns.cs

```diff
@@ -0,0 +1,186 @@
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.Extensions.Logging;
+
+namespace DurableTest
+{
+    public class Patterns
```

Comment:
Thoughts on creating automation to run these new smoke tests?

Reply:
That would be great! Are these tests not run as part of [this smoke test](https://github.com/Azure/azure-functions-durable-extension/blob/8dd369afe54793f9d8c2702c6cc7a06f88cfdbf0/.github/workflows/smoketest-dotnet-isolated-v4.yml)? I thought it would be picked up just by looking at the file path that's used for the test, but maybe I'm missing something.

Reply:
I am actually unfamiliar with the test. I can move these if we want. I added them because I was verifying how more complex orchestration calls behave with distributed tracing here.

Reply:
I removed this file for now, we can add it in a separate PR if we want more advanced tests.